src/main/java/com/amazonaws/services/dynamodbv2/datamodeling/AttributeEncryptor.java

```diff
@@ -58,9 +64,20 @@ public DynamoDBEncryptor getEncryptor() {
     public Map<String, AttributeValue> transform(final Parameters<?> parameters) {
         // one map of attributeFlags per model class
         final ModelClassMetadata metadata = getModelClassMetadata(parameters);
+
+        final Map<String, AttributeValue> attributeValues = parameters.getAttributeValues();
```

Comment:
Could you please add a short comment explaining the case being handled here / why this is set to behave this way, for clarity/maintainers?